.storybook/preview.tsx

```diff
@@ -8,7 +8,7 @@ import "../src/styles/globals.css";
 
 const preview: Preview = {
   parameters: {
-    actions: { argTypesRegex: "^on[A-Z].*" },
+    // actions: { argTypesRegex: "^on[A-Z].*" },
```

Comment:
we should do something with this other than leave it commented out

Reply:
If our stories/tests are working/passing as expected with it commented out, then I think we can just remove it. This global regexp-based configuration has been deprecated and needs to be configured in stories as needed:

https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#implicit-actions-can-not-be-used-during-rendering-for-example-in-the-play-function